USER_GUIDE.md

```diff
@@ -97,101 +98,151 @@ const client = new Client({
-  var index_name = 'books';
-  var settings = {
-    settings: {
-      index: {
-        number_of_shards: 4,
-        number_of_replicas: 3,
-      },
+console.log('Creating index:');
+
+var index_name = 'books';
+var settings = {
+  settings: {
+    index: {
+      number_of_shards: 4,
+      number_of_replicas: 3,
     },
-  };
+  },
+};
 
-  var response = await client.indices.create({
-    index: index_name,
-    body: settings,
-  });
+var response = await client.indices.create({
+  index: index_name,
+  body: settings,
+});
 
-  console.log(response.body);
+console.log(response.body);
 ```
 
 ## Add a Document to the Index
 
 ```javascript
-  console.log('Adding document:');
-
-  var document = {
-    title: 'The Outsider',
-    author: 'Stephen King',
-    year: '2018',
-    genre: 'Crime fiction',
-  };
-
-  var id = '1';
-
-  var response = await client.index({
-    id: id,
-    index: index_name,
-    body: document,
-    refresh: true,
-  });
+console.log('Adding document:');
+
+var document = {
+  title: 'The Outsider',
+  author: 'Stephen King',
+  year: '2018',
+  genre: 'Crime fiction',
+};
+
+var id = '1';
+
+var response = await client.index({
+  id: id,
+  index: index_name,
+  body: document,
+  refresh: true,
+});
 
-  console.log(response.body);
+console.log(response.body);
 ```
 
 ## Search for the Document
 
 ```javascript
-  console.log('Search results:');
-
-  var query = {
-    query: {
-      match: {
-        title: {
-          query: 'The Outsider',
-        },
+console.log('Search results:');
+
+var query = {
+  query: {
+    match: {
+      title: {
+        query: 'The Outsider',
       },
     },
-  };
+  },
+};
 
-  var response = await client.search({
-    index: index_name,
-    body: query,
-  });
+var response = await client.search({
+  index: index_name,
+  body: query,
+});
 
-  console.log(response.body.hits);
+console.log(response.body.hits);
 ```
 
 ## Delete the document
 
 ```javascript
-  console.log('Deleting document:');
+console.log('Deleting document:');
 
-  var response = await client.delete({
-    index: index_name,
-    id: id,
-  });
+var response = await client.delete({
+  index: index_name,
+  id: id,
+});
 
-  console.log(response.body);
+console.log(response.body);
 ```
 
 ## Delete the index
 
 ```javascript
-  console.log('Deleting index:');
+console.log('Deleting index:');
+
+var response = await client.indices.delete({
+  index: index_name,
+});
+
+console.log(response.body);
+```
+
+## Create a Point in Time
+
+```javascript
+console.log('Creating a PIT:');
+
+var response = await client.createPit({
+  index: 'books*',
+  keep_alive: '100m',
+  expand_wildcards: 'all',
+});
+
+console.log(response.body);
+```
+
+## Get all PITs
+
+```javascript
+console.log('Getting all PITs:');
+
+var response = await client.getAllPits();
+
+console.log(response.body);
+```
+
+## Delete a Point in Time
+
+```javascript
+console.log('Deleting a PIT:');
+
+var response = await client.deletePit({
```

Comment:
Could you add a comment here for how a user can get this `pit_id` ?

Reply:
I'm not sure if I understand this completely. A particular point in time (PIT) is denoted by a `pit_id`. It's like a particular index is denoted by `index_name`

Reply:
How do we get a pit_id?

Reply:
We get it when we create a point in time.